Makefile

```diff
@@ -20,7 +20,7 @@ PROTO_GIT_URL = $(DRONE_PROTO_GIT_URL)
 endif
 
 # Accept a branch from which we need to checkout proto files. Useful for dev testing.
-PROTO_BRANCH ?= master
+PROTO_BRANCH ?= producer_base
```

Comment:
I dont the variable `PROTO_BRANCH` is being used anywhere. This can be dropped.

Reply:
Removed.